db/migrate/20200523233032_create_sessions.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSessions < ActiveRecord::Migration[6.0]
+  def change
+    create_table :sessions do |t|
+      t.references :interview, null: false, foreign_key: true
+      t.references :participant, null: false, foreign_key: true
```

Comment:
We can have role to define what is the participant role?

Reply:
Sir, I have added the 'participant role and accommodated necessary changes in 'crud' branch.
![Screenshot from 2020-05-27 22-51-18](https://user-images.githubusercontent.com/65555640/83052564-23292580-a06d-11ea-901b-5eaee223da3c.png)

Reply:
This PR is `crud` branch itself. I can't find it in the schema file or any migration for that?